src/bundler/graph.zig

```diff
@@ -0,0 +1,128 @@
+pub const Graph = struct {
```

Comment:
This should be a top-level struct instead.


```suggestion
const Graph = @This();
```